### PR TITLE
Add logo, name, url for PolkaFoundry

### PR DESCRIPTION
**Pull request by thanhtung6824**
This PR aims to add incoming name, logo, and URL for PolkaFoundry. Please let me know if I made any mistakes. Thank you

### CODE REVIEW

File: packages/apps-config/src/ui/logos/index.ts [hunk 1]
```diff
@@ -248,6 +252,7 @@ export const namedLogos: Record<string, unknown> = {
   polkabtc: nodePolkaBTC,
   polkadex: nodePolkadex,
   polkadot: nodePolkadot,
+  polkafoundry: nodePolkaFoundry,
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:255]: You would just need an entry for `rococoPolkaFoundry: nodePolkaFoundry` as well (the `info` in the RPC endpoint matches on this name)

Comment (thanhtung6824) [reply packages/apps-config/src/ui/logos/index.ts:255]: @jacogr TY. I just added it.

**Review by jacogr — COMMENTED** [review]
Just run yarn lint locally - have not checked CI, but it has a small complaint ;)

**Comment by thanhtung6824** [comment]
@jacogr Thank you. I fixed it

**Review by jacogr — APPROVED** [review]
Thank you.

**Comment by thanhtung6824** [comment]
@jacogr 
image
The color is not affected. Maybe I need to change the name to match the info in the RPC endpoint? In this case is "Polkafoundry PC1" instead of "Polkafoundry"

**Comment by jacogr** [comment]
The colors, as mentioned in the code is based on there the `system.chain` RPC result or the actual software name via the `system.name` RPC.

Best is always to check it locally.

**Pull request merged**